src/pages/index.js

```diff
@@ -9,8 +9,13 @@ import Landing from "../sections/Landing";
 import About from "../sections/About";
 import Team from "../sections/Team";
 import Layout from "../components/Layout";
+import configureDarkMode from "../libs/configureDarkMode";
+
 
 const IndexPage = () => {
+
+    configureDarkMode();
```

Comment:
I'm just thinking not tried but is it working when we go to another page? It looks like it is working on index page but what happens if we go to download page? Is it still works because it looks like you are just adding `dark` class on index page

Reply:
Good catch, I didn't even think about that. It seems to maintain state when navigating to the download page, which makes sense, but then upon reload defaults to light. I added the method to all pages and now it works fine, no flash and respects system theme, for all of them (even upon changing in system settings and navigating to the page.)